ui/tests/integration/components/sync/secrets/page/overview-test.js

```diff
@@ -172,49 +172,28 @@ module('Integration | Component | sync | Page::Overview', function (hooks) {
     test('it should show dismiss banner', async function (assert) {
       await this.renderComponent();
 
-      assert.dom(overview.optInDismiss).exists('dismiss opt-in banner shows');
+      assert.dom(overview.optInBanner.dismiss).exists('dismiss opt-in banner shows');
     });
 
     test('it should navigate to the opt-in modal', async function (assert) {
       await this.renderComponent();
 
-      await click(overview.optInBannerEnable);
+      await click(overview.optInBanner.enable);
 
-      assert.dom(overview.optInModal).exists('Opt-in modal is shown');
-      assert.dom(overview.optInConfirm).isDisabled('Confirm button is disabled when checkbox is unchecked');
-
-      await click(overview.optInCheck);
-      assert.dom(overview.optInConfirm).isNotDisabled('confirm button is enabled once checkbox is checked');
-    });
-
-    test('it should make a POST to activate the feature', async function (assert) {
-      assert.expect(1);
-
-      await this.renderComponent();
-
-      this.server.post('/sys/activation-flags/secrets-sync/activate', () => {
-        assert.true(true, 'POST to secrets-sync/activate is called');
-        return {};
-      });
-
-      await this.renderComponent();
-
-      await click(overview.optInBannerEnable);
-      await click(overview.optInCheck);
-      await click(overview.optInConfirm);
```

Comment:
all of this is now tested within the [modal component tests](https://github.com/hashicorp/vault/pull/26822/files#diff-b0980042b7008b306e7a93c74b8f82c1314d1c9c261e2a50d67a64549fe9009cR75-R110) ✨